src/attribute-manager.cpp

```diff
@@ -19,6 +19,7 @@ AttributeManager &AttributeManager::instance()
 
 AttributeManager::AttributeList &AttributeManager::attributes()
 {
+    qSort(m_attributes.begin(), m_attributes.end(), [](const Resource* v1, const Resource* v2){return v1->name() < v2->name();});
```

Comment:
I don't think it makes sense to do this in the getter.
This means it'll be done every time someone wants a reference to the list.